unattended_installer/cert_tool/certFunctions.sh

```diff
@@ -151,6 +153,7 @@ function cert_generateDashboardcertificates() {
             eval "openssl x509 -req -in /tmp/wazuh-certificates/${dashboard_node_names[i]}.csr -CA /tmp/wazuh-certificates/root-ca.pem -CAkey /tmp/wazuh-certificates/root-ca.key -CAcreateserial -out /tmp/wazuh-certificates/${dashboard_node_names[i]}.pem -extfile /tmp/wazuh-certificates/${dashboard_node_names[i]}.conf -extensions v3_req -days 3650 ${debug}"
 
         done
+        cert_setpermisions
```

Comment:
I would change the function call to this place:

https://github.com/wazuh/wazuh-packages/blob/2e1630dd0290742ef43149b5dae2ec90c25d6711/unattended_installer/cert_tool/certMain.sh#L204-L209

Reply:
Done [ebf6847ac4b7d6ddb4a7e469ed570c4ddf3c9f41](https://github.com/wazuh/wazuh-packages/pull/1415/commits/ebf6847ac4b7d6ddb4a7e469ed570c4ddf3c9f41)